### DISCUSSION TITLE
Talents Wanted!!! Join the Revolution: Titan Network & Fansland Unveil the First-ever WEB3 Music Festival Experience!

### DISCUSSION THREAD

**Discussion by MaryTitan** [Titannet-dao/Community#4] [RetroPGF]
💥We are thrilled to announce a groundbreaking collaboration between Titan Network and Fansland (https://fansland.io/bsc) to bring you an innovative WEB3 music festival experience (https://fansland.io/bsc/music-festival). This May, in Thailand, the Fansland Music Festival will debut as the world's first WEB3 music festival, featuring an impressive array of stars, officials, KOLs, cryptocurrency influencers, and project participants. Titan Network is proud to be the chosen decentralized service partner for this festival and is actively seeking your help to develop a cutting-edge live photo streaming feature for Titan Storage.

🔥This feature, underpinned by blockchain and decentralized storage technologies, will be the go-to platform for showcasing live photos from the Fansland Music Festival, ensuring a secure, efficient, and decentralized live photo streaming experience for all attendees, powered by Titan Storage.

🚀The development of this live photo streaming feature marks our first initiative under Titan Network's RetroPGF (https://github.com/Titannet-dao/Community/discussions/3) incentive program. This initiative not only aims to enhance our service offerings but also seeks to inspire and mobilize more developers to join our community and share their unique skills and insights.

📣As a participant in this co-creation effort, you'll gain the chance to:
Engage with an innovative WEB3 project;
Partner with an internationally recognized music festival, allowing you to showcase your technical prowess to a worldwide audience;
Collaborate with professionals from various backgrounds, fostering growth and addressing challenges as a unified team.

👀We're on the lookout for co-creators endowed with the following skills and attributes:
Devoted developers, product managers, strategists, and designers eager to forge meaningful technology-driven experiences;
Individuals proficient in both front-end and back-end development, capable of crafting stable and efficient systems;
Enthusiasts with a solid understanding of blockchain technology and a readiness to work within a decentralized framework;
Team-oriented individuals with a proactive attitude, an openness to share knowledge, and a zeal for acquiring new insights;
Passionate advocates for music, art, and pioneering technology.

🛡Project Deadline:
All contributions should be finalized by April 20, 2024.

💰Rewards:
Contributors will be compensated for their efforts in US dollars or receive a share of Titan's growth profits, in accordance with the RetroPGF program guidelines.

🌈If you're drawn to this initiative and eager to contribute to the WEB3 music festival, we warmly invite you to join us. For those interested, please leave a message or reach out through our official Telegram (TitanBlockchain) or email (titan@titannet.io) to discuss the collaboration in more detail. Thank you!

Warm regards,
Titan Network

🛠About Titan Network:
Titan Network is a pioneering digital resource network dedicated to aligning the untapped resources of our community with global demand, within a decentralized and open-source framework. We deliver functionalities akin to traditional cloud services, including hot storage and CDN acceleration. Titan Network has introduced Titan Storage, a showcase decentralized storage solution with fully open-source code. Moving forward, Titan Network aims to expand this project to cater to both Web2 and Web3 users, driven by the RetroPGF reward mechanism to stimulate community developer engagement. Users benefit from hot storage options and rapid downloads from the closest Titan node worldwide, supplemented by multiple backups for enhanced data security. These backups are also secured on Filecoin, a leading decentralized storage platform, within 48 hours to guarantee data safety, capitalizing on Filecoin's robust brand trust and its secure data protection measures. Filecoin storage providers (SPs) commit to staking fees to ensure data integrity throughout the contract term, with significant penalties from the Filecoin network for any data breaches. For our users, accessing these advanced services incurs minimal cost, providing a seamless solution for utilizing Filecoin without the hassle of direct engagement with storage providers or navigating complex data management processes. With straightforward operations, users can tap into a global hardware resource pool worth billions for easy and efficient data storage.

--------------------------------------------------------------------------------------------------------------------------------------
🎊Thank you all for getting engaged in this event. We have received many awesome proposals and are now announcing an early closure since we found the right solution for the demand. Please stay tuned for upcoming events and news. For talents who are interested in joining us, please refer to our [RetroPGF program](https://github.com/Titannet-dao/Community/discussions/3) and don't hesitate to reach out to us!

2nd, April
Titan Network

**Comment by iamharrisonn** [comment]
I have ten years of experience as a product manager. I can give planning this project a try : )

**Reply by MaryTitan** [reply]
Great! Please reach out to me on TG(ID: TitanBlockchain), or what is your TG ID?

**Reply by MaryTitan** [reply]
Hi there! I am very glad that you reached out to us. 
The Functionalities to be developed include:
1. Automate the collection of data from the image platform utilized by Fansland, specifically enabling mass photo downloads.
2. Perform mass uploads to a designated Titan Storage account, creating links for external image access.
3. Develop the front-end for a photo live streaming page, ensuring compatibility with both PC and mobile devices (including H5 support).
4. Feature photos from the Fansland Music Festival on the live streaming page.

Let me know if you have any questions or your ideas. Thank you.
Looking forward to hearing from you!

**Comment by yann-y** [comment]
Similar to cloud album functionality? I am interested in how to develop applications on Titan network

**Reply by MaryTitan** [reply]
Great! Please reach out to me on TG(ID: TitanBlockchain), or what is your TG ID?

**Comment by xiiiAtCn** [comment]
I  have worked as a front end developer with 8 years. I can help build this project.

**Reply by MaryTitan** [reply]
Great! Please reach out to me on TG(ID: TitanBlockchain), or what is your TG ID?

**Reply by MaryTitan** [reply]
Hi there! I am very glad that you reached out to us.
The Functionalities to be developed include:

Automate the collection of data from the image platform utilized by Fansland, specifically enabling mass photo downloads.
Perform mass uploads to a designated Titan Storage account, creating links for external image access.
Develop the front-end for a photo live streaming page, ensuring compatibility with both PC and mobile devices (including H5 support).
Feature photos from the Fansland Music Festival on the live streaming page.
Let me know if you have any questions or your ideas. Thank you.
Looking forward to hearing from you!

**Reply by xiiiAtCn** [reply]
TG ID: lungern (My TG is temporarily restricted. I cant reach you now)

**Comment by Carl302** [comment]
10年win平台C#开发程序员  熟悉PC win平台兼容性问题

**Reply by MaryTitan** [reply]
Great! Please reach out to me on TG(ID: TitanBlockchain), or what is your TG ID?

**Comment by ljx2099** [comment]
lfg！

**Reply by MaryTitan** [reply]
Can you please share your ideas, thx! Please reach out to me on TG(ID: TitanBlockchain), or what is your TG ID?
